code/game/objects/items/handcuffs.dm

```diff
@@ -57,7 +58,7 @@
 								"<span class='userdanger'>[user] is trying to put [src.name] on you!</span>")
 
 			playsound(loc, cuffsound, 30, 1, -2)
-			if(do_mob(user, C, 40) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore()))
+			if(do_mob(user, C, t) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore())) //austation -- Makes handcuffing paralzyed people faster
```

Comment:
Could have applied it to the t variable instead but this has less austation vs bee line diffs :)
```suggestion
			if(do_mob(user, C, (C.has_status_effect(STATUS_EFFECT_PARALYZED) ? 20 : 40)) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore())) //austation -- Makes handcuffing paralzyed people faster
```